### PR TITLE
chore: added types for few utility methods

### PR DESCRIPTION
**Pull request by anikethsaha**
- [ ] This is a **bugfix**
- [ ] This is a **feature**
- [ ] This is a **code refactor**
- [ ] This is a **test update**
- [ ] This is a **docs update**
- [x] This is a **metadata update**

### For Bugs and Features; did you add new tests?
NA

### Motivation / Use-Case
Follow up of https://github.com/webpack/webpack-dev-server/pull/2107

### Breaking Changes
NA

### Additional Info
Added types for 
```bash
lib/utils/colors.js
lib/utils/createCertificate.js
lib/utils/createConfig.js
```
and added these files in `tsconfig`, will change the `include` in `tsconfig` to `lib/utils/*` once all of the files there is completed

### CODE REVIEW

**Comment by evilebottnawi** [comment]
Need fix CI problems :+1:

**Comment by anikethsaha** [comment]
Not sure with where/why CI is failing. Need help with it !

File: lib/utils/colors.js [hunk 1]
```diff
@@ -1,6 +1,12 @@
 'use strict';
 
 const colors = {
+  /**
+   * info color Method
+   * @param {?boolean} useColor - use color with your message
+   * @param {string} msg - Your log message
```
Comment (hiroppy) [inline lib/utils/colors.js]: log message

File: lib/utils/colors.js [hunk 2]
```diff
@@ -9,6 +15,12 @@ const colors = {
 
     return msg;
   },
+  /**
+   * error color Method
+   * @param {?boolean} useColor - use color with your message
+   * @param {string} msg - Your log message
```
Comment (hiroppy) [inline lib/utils/colors.js]: ditto

**Comment by anikethsaha** [comment]
@hiroppy done 👍

**Comment by anikethsaha** [comment]
cc @evilebottnawi @hiroppy

**Pull request closed without merging**